src/featureSelectors.test.ts

```diff
@@ -29,13 +29,40 @@ describe("featureSelectors", () => {
         name: "users",
         entity: "users",
       });
+
       const output = getDenormalizedData.resultFunc(featureSlices, true);
+
       expect(output).toEqual({
         users: [
           { id: 1, name: "Garrett" },
           { id: 2, name: "Laura" },
         ],
       });
     });
+
+    it("should return undefined if the selected slice has a falsy status", () => {
+      const featureSlices = {
+        entities: {
+          users: { 1: { id: 1, name: "Garrett" }, 2: { id: 2, name: "Laura" } },
+          animals: { 1: { id: 1, name: "dog" }, 2: { id: 2, name: "cat" } },
+        },
+        users: {
+          data: { users: [1, 2] },
+          status: "",
+        },
+        animals: {
+          data: { animals: [1, 2] },
+          status: "foo",
+        },
+      };
+      const getDenormalizedData = makeGetDenormalizedData({
+        name: "users",
+        entity: "users",
+      });
+
+      const output = getDenormalizedData.resultFunc(featureSlices, true);
```

Comment:
please use a constant for each value passed to the selector. it helps improve the readability of the selector, especially when you're using `resultFunc`